playbooks/cloud-provider-openstack-acceptance-test-standalone-cinder/run.yaml

```diff
@@ -46,8 +46,8 @@
           export CLOUD_CONFIG=/etc/kubernetes/cloud-config
 
           # location of where the kubernetes processes log their output
-          mkdir -p '{{ ansible_user_dir }}/workspace/logs/kubernetes'
-          export LOG_DIR='{{ ansible_user_dir }}/workspace/logs/kubernetes'
+          mkdir -p '{{ k8s_log_dir }}'
+          export LOG_DIR='{{ k8s_log_dir }}'
```

Comment:
- we can add a task into k8s base job to ensure `{{ k8s_log_dir }}`, do need to mkdir in every run.yaml
- Please use `{{ k8s_log_dir }}` in following log path, do not export to `LOG_DIR`

Reply:
- A new role will be created to avoid executing `mkdir` in every run.yaml in next PR, no need to change here
- LOG_DIR env is needed during setup the k8s services with a default value of /tmp
https://github.com/kubernetes/kubernetes/blob/master/hack/local-up-cluster.sh#L232

Reply:
Thank you clarifying these.